tests/options/no_color.rs

```diff
@@ -0,0 +1,22 @@
+use crate::common::TestDir;
+
+#[test]
+fn output_without_colors() {
```

Comment:
This function doesn't test `--no-color` option.

Reply:
@Nikhil0487, @dotenv-linter/core Do you have any ideas how we can test `--no-color` option?

Reply:
The only thing that came to my mind is an integration test and match the output from the tool with the correct ANSI escape sequences. see: https://www.shellhacks.com/bash-colors/

Reply:
@Nikhil0487 Can you try this way?